server/src/handler_upload.c

```diff
@@ -229,8 +245,7 @@ onion_connection_status handler_remove_engine(void *_, onion_request *req,
 		if (name == NULL || plugin_is_unremovable(name)) {
 			syslog_server(LOG_ERR, 
 			              "Request: Remove engine - engine name is invalid or engine is unremovable", 
-						  name);
-						  
+			              name);
 			onion_response_printf(res, "Engine name is invalid or engine is unremovable");
 			onion_response_set_code(res, HTTP_BAD_REQUEST);
```

Comment:
Swap with line above

Reply:
not sure what that means, I think "name" is not supposed to be here at all - will remove that param and associated line.

Reply:
`"Request: Remove engine - engine name \"%s\" is invalid or engine is unremovable",`
`(name == NULL) ? "null" : name`